### DISCUSSION TITLE
Menüpunkt Homematic IP Geräte anlernen fehlt

### DISCUSSION THREAD

**Discussion by maethner** [OpenCCU/OpenCCU#2725] [🇩🇪 Fragen&Antworten] [answered]
Ich betreibe Rasberrymatic in einer VM auf einem  Synology NAS. Als Funkmodule nutze ich zwei LAN Gateway HM-LGW-O-TW-W-EU, die schon vorhanden waren.

Als ich den digitale Stromzähler mit LED-Schnittstelle, (HmIP-ESI-LED) einbinden wollte, fehlte auf einmal das entsprechende Menue in der Oberfläche. 

Version: 3.75.7.20240420 (OVA).

P.S.: Keine Ahnung, ob das damit zusammenhängt, aber der Dutycircle auf einem LAN-Gateway ist relativ hoch (oft über 25%)

![Anlernen](https://github.com/jens-maus/RaspberryMatic/assets/154928327/84483c11-6b1c-432e-a2f2-f17f330ee6f8)

Vielen Dank für jede Hilfe!

**Comment by jens-maus** [comment, marked as answer]
Na wenn du nur zwei `HM-LGW-O-TW-W-EU` für die Homematic Kommunikation hast, kann deine CCU bzw. RaspberryMatic eben auch nur mit den alten BidCos-RF/HomeMatic Geräten kommunizieren. Für homematicIP Kommunikation benötigst du natürlich ein Funkmodul/Coprozessor wie das `RPI-RF-MOD` oder `HmIP-RFUSB`. Mehr Infos zu den Vorrausetzungen findest du in der Dokumentation (siehe https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen) in der dort sichtbar großen Tabelle.

**Reply by maethner** [reply]
Hallo Jens!

Whow. Tausend Dank für die schnelle und vor allem problemlösende Antwort.

Stick ist geordert. Merci!
